samples/bluetooth/direct_test_mode/src/dtm.c

```diff
@@ -66,15 +66,18 @@
 #define WAIT_TIMER_IRQ_HANDLER     NRFX_CONCAT_3(nrfx_timer_,		 \
 						 WAIT_TIMER_INSTANCE,    \
 						 _irq_handler)
+/* Note that the timer instance 2 is used in the FEM driver. */
+
+#ifdef NRF52840_XXAA
```

Comment:
I think that we widely used `CONFIG_SOC_NRF52840` instead of `NRF52840_XXAA`. Could you switch ifdefs to it ?

Reply:
I think this should be done in a different PR as a separate task. The whole DTM sample needs to be aligned if we decide to go for CONFIG_SOC_ definitions. Also, we have other places where we use non-Kconfig like a nrf21540.c source file:

#if defined(NRF5340_XXAA_NETWORK)